package.json

```diff
@@ -4,8 +4,9 @@
   "description": "A temporary thing to get stuff on the npm homepage",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "node build.js > data.json"
+    "//pretest": "standard",
```

Comment:
Need to add uncomment this (I didn't even know you could do this) and fix some style errors.